packages/sku/scripts/postinstall.js

```diff
@@ -30,7 +30,7 @@ if (packageJsonExists) {
   const skipPostInstall = skuSkipPostInstall || skuSkipPostinstall;
   const hasSku =
     Boolean(devDependencies?.sku) ||
-    // TODO: get rid of this part when we remove treat
+    // sku should always be a dev dependency, but some repos may still have it as a regular dependency
```

Comment:
Should we warn them? It would somewhat reduce the size of their Docker image 🌍🔥

Reply:
That's a good idea. More warning banners 🎉

Reply:
![jokes](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.ytimg.com%2Fvi%2FEC4O1g0ux20%2Fmaxresdefault.jpg&f=1&nofb=1&ipt=34f350a90edcbaf561acc92fc386373b6d00dc2467e0c362664daea21261af1c&ipo=images)

Reply:
https://github.com/seek-oss/sku/pull/809/commits/64c4d37903975c73d8acdd8c5c58b44bcfc44a41